lib/export-profiles.js

```diff
@@ -42,10 +42,10 @@ function ExportProfiles(api, logger) {
 
       var params = _ref.params;
 
-      var query = params && params.templateIds ? '?' + templateIds.map(function (id) {
+      var query = params && params.templates ? '?' + params.templates.map(function (id) {
```

Comment:
I haven't looked too deeply here, but are we changing the external API with this?